src/containers/sortable-wrapper/SortableWrapper.tsx

```diff
@@ -0,0 +1,43 @@
+import { CSSProperties, FC } from 'react'
+import { useSortable } from '@dnd-kit/sortable'
+import { CSS } from '@dnd-kit/utilities'
+import { Box, SxProps } from '@mui/material'
```

Comment:
Use default import of `Box` instead of named

Reply:
didn't notice, thank you. Fixed